sphinx_gallery/tests/test_full.py

```diff
@@ -26,11 +26,13 @@
 
 import pytest
 
-N_TOT = 13  # examples (plot_*.py in examples/**)
-
+# file inventory for tinybuild:
+N_EXAMPLES = 13  # total number of plot_*.py files in tinybuild/examples
 N_FAILING = 2
-N_GOOD = N_TOT - N_FAILING
-N_RST = 17 + N_TOT + 1  # includes module API pages, etc.
+N_GOOD = N_EXAMPLES - N_FAILING  # galleries that run w/o error
+N_INDEX = 2 + 2 + 1  # indices SG generates + execution times + local_module
+N_OTHER = 9 + 2 + 2  # gen_modules + mainindex.rst + minigallery.rst + 2??
+N_RST = N_EXAMPLES + N_INDEX + N_OTHER  # includes module API pages, etc.
```

Comment:
This looks to work, but I don't know why I need the +2 here.  sphinx is mysterious!